_example/default_subnet/outputs.tf

```diff
@@ -1,34 +1,34 @@
 output "subnet_name" {
-  description = "The name of the subnet."
   value       = module.subnet.default_subnet_name
+  description = "The name of the subnet."
 }
 
 output "subnet_id" {
-  description = "The subnet ID."
   value       = module.subnet.default_subnet_id
+  description = "The subnet ID."
 }
 
 output "resource_group_name" {
-  description = "The name of the resource group in which the subnet is created in."
   value       = module.resource_group.resource_group_name
+  description = "The name of the resource group in which the subnet is created in."
 }
 
 output "virtual_network_name" {
+  value       = module.vnet.vnet_name
   description = "The name of the virtual network in which the subnet is created in."
-  value       = join("", module.vnet.vnet_name)
 }
 
 output "address_prefixes" {
-  description = "The address prefixes for the subnet."
   value       = module.subnet.default_subnet_address_prefixes
+  description = "The address prefixes for the subnet."
 }
 
 output "route_table_id" {
-  description = "The Route Table ID."
   value       = module.subnet.route_table_id
+  description = "The Route Table ID."
 }
 
 output "route_table_associated_subnets" {
+  value       = module.subnet[*].route_table_associated_subnets[0]
```

Comment:
why do we need to specify [*] and [0] here in output? Can we not directly use it?